src/Uno.Extensions.Templates/content/unoapp-extensions/Directory.Build.nocpm.props

```diff
@@ -1,6 +1,6 @@
 <Project ToolsVersion="15.0">
 	<PropertyGroup>
-		<LangVersion>10.0</LangVersion>
+		<LangVersion>latest</LangVersion>
```

Comment:
Change to latest known good version, which is 11. Concern with using latest is that it may cause issues if there are unsupported features introduced